src/MusicStore/test/MusicStore.E2ETests/MusicStore.E2ETests.csproj

```diff
@@ -11,6 +11,7 @@
     <NoWarn>$(NoWarn);NU1605</NoWarn>
     <!-- https://github.com/aspnet/AspNetCore/issues/6549 -->
     <BuildHelixPayload>false</BuildHelixPayload>
+    <IsTestProject>false</IsTestProject>
```

Comment:
Put a comment in?

Reply:
I have the follow ready - https://github.com/aspnet/AspNetCore/pull/8482. Hopefully this doesn't stay skipped very long

Reply:
For future reference, comments would have helped. It's May 31 and this test is still disabled, and it took me some digging to find the reason.

Reply:
Oops